ONSdigital/response-operations-ui
# Investigate jQuery and resolve where possible

# What and why?
There is a vulnerability with the package https://code.jquery.com/jquery-3.3.1.min.js. An investigation was conducted with the aim of removing this and any other jQuery references. This achieves that.

# How to test?
- Run unit tests
- Run acceptance tests
- As the base.html template has been affected, check all pages to see if they are correctly working

# Jira
https://jira.ons.gov.uk/browse/RAS-1235
